### PR TITLE
Add Dangerfile

### PR DESCRIPTION
**Pull request by MortyMerr**
https://github.com/RxSwiftCommunity/RxSwiftExt/issues/142
I am open for discussion about Dangerfile. Sadly, I failed to found generic example in RxSwiftCommunity

### CODE REVIEW

**Comment by MortyMerr** [comment]
Hey @freak4pc sorry for a ping. Could u please provide your opinion about this PR? Can we proceed with this Dangerfile?

File: Dangerfile [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# set the number of lines that must be changed before this classifies as a 'Big PR'
+@SDM_DANGER_BIG_PR_LINES = 50
```
Comment (freak4pc) [inline Dangerfile]: what does the SDM prefix mean in this context?

File: Dangerfile [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+# Fail if changes to immutable files, such as License or CoC
+fail('Do not modify the license or Code of Conduct') if did_modify(@SDM_DANGER_IMMUTABLE_FILES)
+
+# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+warn("PR is classed as Work in Progress") if github.pr_title.include? "[WIP]"
+
+# Warn when there is a big PR
+warn("Big PR") if git.lines_of_code > @SDM_DANGER_BIG_PR_LINES
+
+
```
Comment (freak4pc) [inline Dangerfile]: remove extra l ine

Comment (MortyMerr) [reply Dangerfile]: Thanks, fixed

File: Dangerfile (hunk 1 above)
Comment (MortyMerr) [reply Dangerfile]: I don't know :(
I thought it's just a variable name, copied from here https://github.com/samdmarshall/danger/blob/master/Dangerfile

Comment (MortyMerr) [reply Dangerfile]: Honestly, I don't know. Just saw that naming many times in other repos.
https://github.com/search?q=SDM_DANGER_BIG_PR_LINES&type=Code

Comment (freak4pc) [reply Dangerfile]: Probably more Dangerfiles without it :) It doesn't make much sense, I'd remove it

Comment (MortyMerr) [reply Dangerfile]: Removed, thanks

**Comment by MortyMerr** [comment]
@freak4pc yes it does. It was a temporary solution just to check everything is ok. I am not an admin of RxSwiftComminuty so I can't add tokens for org.

**Pull request closed without merging**